<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginClass</class>
 <widget class="QDialog" name="LoginClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QLineEdit" name="ledtPWD">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>250</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labicon">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>20</y>
     <width>160</width>
     <height>127</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtLogin">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>310</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>登 录</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cboAccount">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>190</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="maxVisibleItems">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="labVersion">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>370</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="login.qrc"/>
 </resources>
 <connections/>
</ui>
